### PR TITLE
Docker secretArgs

### PR DESCRIPTION
**Pull request by jilleJr**
New fields added to the Docker step type.

- Added docs on Docker secret args
- Updated wharf-ci-schema.json

### CODE REVIEW

File: docs/usage-wharfyml/step-types/docker.md [hunk 1]
```diff
@@ -18,4 +18,8 @@ docker:
   args: # dockerfile building ARGs
     - FIRST_ARG=Value from '.wharf-ci.yml' file!
     - SECOND_ARG=2147483647+1
+  secretName: mysecret # if specified, the target Kubernetes secret name will be mounted to be used by the secretArgs setting. The secret must have the name 'wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-mysecret'.
```
Comment (Alexamakans) [inline docs/usage-wharfyml/step-types/docker.md:21]: Nit: Keep consistent with `wharf-ci-schema.json`

```suggestion
  secretName: mysecret # if specified, the target Kubernetes secret name will be mounted to be used by the secretArgs setting. The secret must have the name 'wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-${secretName}'.
```

File: docs/_static/wharf-ci-schema.json [hunk 2]
```diff
@@ -250,6 +250,27 @@
                                     "SECOND_ARG=12453125"
                                 ]
                             ]
+                        },
+                        "secretName": {
+                            "type": "string",
+                            "description": "If specified, the target Kubernetes secret name will be mounted to be available for secret arguments via the secretArgs setting. The secret must have the name `wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-${secretName}`.",
+                            "default": null,
+                            "examples": [
+                                "myproject-system-tests-secret"
```
Comment (Alexamakans) [inline docs/_static/wharf-ci-schema.json:259]: Nit: Keep consistent with `docker.md` example.

```suggestion
                                "mysecret"
```

**Review by Alexamakans — APPROVED** [review]
👍🏻

**Review by fredx30 — APPROVED** [review]
Nice!

**Pull request merged**